src/log/spec.cr

```diff
@@ -97,12 +97,12 @@ class Log
     end
 
     # Validates that at some point the indicated entry was emitted
-    def check(level : Severity, message : String, file = __FILE__, line = __LINE__) : self
+    def check(level : Severity, message : String, file : String = __FILE__, line : Int32 = __LINE__) : self
```

Comment:
Could the other next and check methods with the same params have the type restrictions added to them?